tests/importance_tests/pedanova_tests/test_scott_parzen_estimator.py

```diff
@@ -35,8 +35,7 @@ def test_init_scott_parzen_estimator(dist_type: str) -> None:
             else CategoricalDistribution(choices=["a" * i for i in range(counts.size)])
         ),
         counts=counts,
-        consider_prior=False,
-        prior_weight=0.0,
+        prior_weight=1.0,
```

Comment:
I believe this part should still test `prior_weight=0.0` to check whether PED-ANOVA works with `prior_weight==0.0`.